src/vl-map-wfs-layer.js

```diff
@@ -44,11 +44,13 @@ export class VlMapWfsLayer extends VlMapVectorLayer {
   }
 
   __createSource() {
-    return new OlVectorSource({
+    const olVectorSource = new OlVectorSource({
       format: this.__sourceFormat,
       strategy: this.__loadingStrategy,
       url: this.__getWfsUrl.bind(this),
     });
+    olVectorSource.on('featuresloadend', () => this.rerender());
```

Comment:
@pascalriquier bij de Voortoets hebben jullie hier geen last van? Wijziging werd door @SBylemans uitgevoerd omdat ze merken dat er rerender nodig is nadat features geladen zijn van een WFS laag.

Reply:
Idem bij tiled WMS, image WMS en WMTS kaartlagen (zie ook changes hier).

Reply:
Voorlopig besloten de rerender logica te verwijderen aangezien het applicatief toch niet nodig blijkt te zijn bij WMS. Het probleem dat zich bij WFS zou stellen kan niet gereproduceerd worden op webcomponent niveau dus de oplossing is potentieel overbodig en in het nadeel van andere afnemers.